tikv-configuration-file.md

```diff
@@ -1838,21 +1836,37 @@ Suppose that your machine on which TiKV is deployed has limited resources, for e
++ The interval at which the locally cached timestamps are updated.
++ At an interval of `renew-interval`, TiKV starts a batch of timestamp refresh and adjusts the number of cached timestamps according to the timestamp consumption in the previous period and the [`alloc-ahead-buffer`](#alloc-ahead-buffer-new-in-v640). If you set this parameter to too large a value, the latest TiKV workload changes are not reflected in time. If you set this parameter to too small a value, the load of PD increases. If the write traffic is strongly fluctuating, if timestamps are frequently exhausted, and if write latency increases, you can set this parameter to a smaller value. At the same time, you should also consider the load of PD.
 + Default value: `"100ms"`
 
 ### `renew-batch-min-size`
 
-+ The minimum number of locally cached timestamps.
-+ TiKV adjusts the number of cached timestamps according to the timestamp consumption in the previous period. If the usage of locally cached timestamps is low, TiKV gradually reduces the number of cached timestamps until it reaches `renew-batch-min-size`. If large bursty write traffic often occurs in your application, you can set this parameter to a larger value as appropriate. Note that this parameter is the cache size for a single tikv-server. If you set the parameter to too large a value and the cluster contains many tikv-servers, the TSO consumption will be too fast.
++ The minimum number of a timestamp request.
++ TiKV adjusts the number of cached timestamps according to the timestamp consumption in the previous period. If the requirement of TSO is low, TiKV reduces the request number of each TSO until it reaches `renew-batch-min-size`. If large bursty write traffic often occurs in your application, you can set this parameter to a larger value as appropriate. Note that this parameter is the cache size for a single tikv-server. If you set the parameter to too large a value and the cluster contains many tikv-servers, the TSO consumption will be too fast.
 + In the **TiKV-RAW** \> **Causal timestamp** panel in Grafana, **TSO batch size** is the number of locally cached timestamps that has been dynamically adjusted according to the application workload. You can refer to this metric to adjust `renew-batch-min-size`.
 + Default value: `100`
+
+### `renew-batch-max-size` <span class="version-mark">New in v6.4.0</span>
+
++ The maximum number of a timestamp request.
++ In a default TSO physical time update interval, PD provides at most 262144 TSOs. When exceeding this number, PD does not handle TSO requests. This configuration item is used to avoid the TSO of PD being consumed which might have an impact on other businesses. If you increase the configuration item to improve high availability, you need to decrease the [`tso-update-physical-interval`](/pd-configuration-file.md#tso-update-physical-interval) at the same time to get enough TSO.
```

Comment:
The Chinese does not have `to improve high availability`.
```suggestion
+ In a default TSO physical time update interval, PD provides at most 262144 TSOs. When requested TSOs exceed this number, PD does not handle new TSO requests. This configuration item is used to avoid the exhaustion of TSOs and the reverse impact on other businesses. If you increase the value of this configuration item to improve high availability, you need to decrease the value of [`tso-update-physical-interval`](/pd-configuration-file.md#tso-update-physical-interval) at the same time to get enough TSOs.
```